src/test/base32_tests.cpp

```diff
@@ -2,10 +2,11 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
+#include <string>
```

Comment:
I can reorder them if you think it's worth it.
If you think these contributions are worthless I'll stop.

Reply:
I added a new import and the imports were reorganized as such. If you don't like the order, I can change it, I don't mind, though I would prefer the CI telling me these in the PR instead of wasting time reviewing styles.

Reply:
> I don't understand what you are trying to say.

I'm trying to find out if my contributions are welcome, I've only received hostility so far in return for my invested effort.
It's a decentralized, open source project, I'll contribute what I can and want. If that's not welcome, I'll find a different project where my optimizations are welcome.

Reply:
I was looking for introductory issues, couldn't find any that was a good fit for me.
But just looking at the code I found several optimization opportunities in relevant parts of the code, so I contributed those instead, e.g. [the 4x speedup of base58](https://github.com/bitcoin/bitcoin/pull/29473).